agent/agent.go

```diff
@@ -210,6 +216,8 @@ func NewAgent() *Agent {
 	}
 }
 
+// CreateRootNode create a graph.Node based on the host properties
```

Comment:
CreateRootNode creates a graph.Node based on the host properties and aims to have an unique ID